README.md

```diff
@@ -186,7 +186,7 @@ The following parameters are available for the hiera class:
   This is useful for configuring backend-specific parameters.  
   Default: `''`
 * `keysdir`
-  Directory for hiera to manage for eyaml keys.
+  Directory for hiera to manage for eyaml keys. This is the path where /keys should be managed not the full path to the /keys directory.
```

Comment:
It could be...

where it bites you as you would need to munge that into a new variable to use with 

```
exec { 'createkeys':
        user    => $owner,		        user    => $owner,
        cwd     => $confdir,		 +      cwd     => $keysdir,
```

I was also trying to keep with the pattern used for _keysdir in init.pp